web/admin/app.vue

```diff
@@ -24,11 +24,15 @@ const user = await useUser();
 
 async function login() {
   error.value = null;
+
   const isSignedIn = await auth
     .login(identifier.value, password.value)
     .catch((error) => ({ error }));
 
-  error.value = !isSignedIn;
+  if (isSignedIn.error)
+    error.value = {
+      message: isSignedIn.error
+    };
```

Comment:
**nitpick**: please use full blocks here. with the curly brackets in the error, it can be a bit weird to look at.